examples/minimq-stm32h7/src/main.rs

```diff
@@ -119,7 +117,7 @@ const APP: () = {
         let gpioa = c.device.GPIOA.split(ccdr.peripheral.GPIOA);
         let gpiob = c.device.GPIOB.split(ccdr.peripheral.GPIOB);
         let gpioc = c.device.GPIOC.split(ccdr.peripheral.GPIOC);
-        let gpiof = c.device.GPIOF.split(ccdr.peripheral.GPIOF);
+        let _gpiof = c.device.GPIOF.split(ccdr.peripheral.GPIOF);
```

Comment:
Just remove the GPIO configuration that's not needed. We should ideally have this example be as simply as possible.